src/app/data.service.ts

```diff
@@ -58,4 +58,12 @@ export class DataService {
   public getBand(bandId) : Observable<Band>{
     return this.http.get<Band>('/api/bands/' + bandId);
   }
+
+  public getDeletableJobFamilies() : Observable<JobFamily[]> {
+    return this.http.get<JobFamily[]>('/api/deletableJobFamilies');
+  }
+
+  public deleteJobFamily(jobFamilyId): Observable<Object> {
+    return this.http.delete('/api/deleteJobFamily/' + jobFamilyId);
```

Comment:
Probably too late to be worrying about API schemas at this stage, but for future reference, RESTful APIs don't have verbs in the URL. They're unnecessary, as the verb is already in the request by way of the HTTP verb (see your use of `this.http.delete` in this line of code). So, for example, you could have a single route /api/job-family and use different HTTP verbs such as `GET`, `POST`,` PATCH` and `DELETE` to differentiate between the CRUD operations. 

I'm aware of the time constraints so don't worry about this for now, but bear it in mind for when you're building APIs in the future.

Reply:
I'll keep this in mind for the future, thanks!